src/EditorFeatures/VisualBasicTest/InvertIf/InvertMultiLineIfTests.vb

```diff
@@ -654,5 +654,60 @@ Imports System
 
             Await TestAsync(markup, expected)
         End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)>
+        Public Async Function InvertIfWithoutStatements() As Task
+            Await TestInRegularAndScriptAsync(
+"class C
+    sub M(x as String)
+        [||]If x = ""a"" Then
+        Else
+            DoSomething()
+        End If
+    end sub
+
+    sub DoSomething()
+    end sub
+end class",
+"class C
+    sub M(x as String)
+        If x IsNot ""a"" Then
```

Comment:
it's a bit strange that `=` got replaced with`IsNot` instead of `<>`.

Reply:
I didn't change the logic here... maybe worth a different bug?